wordpress-mobile/WordPress-Android
# Add Latvian to the list of locales included in the app

### Why

A polyglot on the "Making WordPress" .org Slack has been providing and maintaining Latvian translations for the app for more than a year now — with that locale currently being at 97% translation coverage right now [in GlotPress](https://translate.wordpress.org/projects/apps/android/dev/)

> Internal request link: pdnsEh-ev-p2#comment-495

They have rightfully asked us why that locale was not included in the WordPress app, and the reason was simple because we never revised the list of locales our scripts download from GlotPress and include in the project since a very long time and didn't notice that Latvian was now regularly maintained and way over 80% coverage. So it makes sense to finally start including it when building the WordPress app at last.

Since they've asked again (via DM in Slack) today while they made the request 1 year ago and didn't see any change, while this one felt trivial, I decided to quickly do the PR after all.

### How

 - Added an entry for the Latvian locale in the `fastlane/lanes/localization.rb` constant listing all the locales
 - Ran `bundle exec fastlane download_translations` once—which created a commit that updated `available_languages.xml` and all the `values-*/strings.xml` files—then amended that automatic commit to remove all the updated `values-*/` files except `values-lv/`
    - I wanted to avoid disrupting the normal release cycle—where latest translations are only downloaded in the last days of beta-testing after code freeze—hence why I didn't include the other locales' updates, but still wanted to include `values-lv` as this one is brand new and didn't exist in the repo before
    - Besides, including it explicitly in this PR, as well as validating that the `available_languages.xml` file was properly updated by our scripts, will allow you to test the Installable Build associated with this PR in Latvian.

### To Test

 - Install the **WordPress** Installable Build of this PR
 - Open the app, switch to Latvian language in Settings > Language from within the app
 - Check that the app switches to Latvian language and that the app content uses the proper Latvian copies

> **Warning**: I haven't tested this myself (the CI was still running then), so please make sure to test the Installable Build before merging.

> **Note**: As a reminder, the Jetpack app is **intentionally** set up to **only** include the Mag16 languages (see pxLjZ-7b9-p2#comment-19296) and will thus not include Latvian (nor any other non-Mag16). That is why that change will only impact the WordPress app.
